easy_scpi/scpi_instrument.py

```diff
@@ -414,32 +437,29 @@ def _handle_handshake( self ):
                 raise RuntimeError( hs )
 
 
-    def _set_port_windows( self, port ):
+    def _set_port_windows( self, port, match=True ):
         """
         Disconnects from current connection and updates port and id.
         Does not reconnect.
 
         :param port: Name of port to connect to.
         :raises ValueError: If connection type is not specified.
         """
+        prefixes = ['COM', 'USB', 'GPIB', 'TCPIP']
         port_name = port.upper()
 
-        if (
-            ( not port_name.startswith( 'COM' ) ) and
-            ( not port_name.startswith( 'USB' ) ) and
-            ( not port_name.startswith( 'GPIB' ) )
-        ):
-            raise ValueError( "Port must start with 'COM', 'USB', or 'GPIB'." )
+        if not any(port_name.startswith(p) for p in prefixes):
+            raise ValueError( "Port must start with one of the following: %s." % prefixes )

```

Comment:
Please change to a format string.

Reply:
Solved here: https://github.com/bicarlsen/easy-scpi/pull/8/commits/1ba2d04d9f38a76fb996cb3a9776fd7fa4e763e7